charts/kube-prometheus-stack/templates/prometheus/prometheus.yaml

```diff
@@ -332,13 +332,23 @@ spec:
 {{- if .Values.prometheus.prometheusSpec.enforcedNamespaceLabel }}
   enforcedNamespaceLabel: {{ .Values.prometheus.prometheusSpec.enforcedNamespaceLabel }}
 {{- $prometheusDefaultRulesExcludedFromEnforce := (include "rules.names" .) | fromYaml }}
+  # Deprecated, to be removed
```

Comment:
would it be alreadey safe to remove it?

Reply:
would you recommend that? thought not all users update the prometheus-operator version in sync with this chart version and wanted to give them a grace period

Reply:
Everybody coming from a lower major version would also get operator crd version update (or has to do it manually anyway.
Users updating from a minor or patch version should already have newest operator crd version.
Did i miss something?

Imho we could even add the update to v0.56.3.

Reply:
@monotek Implemented your suggestion.

Reply:
Sorry, i haven't thought that trough. Of course that would introduce a breaking change for updating users, so your initial solution was the better one 🤦‍♂️

Reply:
:) ok - reverted